tensorflow_addons/callbacks/tqdm_progress_bar.py

```diff
@@ -186,6 +193,37 @@ def on_batch_end(self, batch, logs={}):
 
                 # update timestamp for last update
                 self.last_update_time = now
+        else:
```

Comment:
actually this else statement is not necessary and may be wrong after thinking about it more.
The reason why we have the logic of updating self.logs as we see examples is that (this is also how the progress bar is doing) we are displaying a `moving average` of metrics over time. If at the end we set `self.logs = log` this means that at the end we are displaying the metrics of last batch instead of the average across the entire epoch/evaluate. I am pretty sure that removing this else statement along with self.logs = logs will still work (since at `on_test_end` you are using `self.logs` and it's already been set.